### DISCUSSION TITLE
Using Sonnet

### DISCUSSION THREAD

**Discussion by ghost** [coin-or/Sonnet#30] [Q&A] [answered]
Hi, 
I've just started on programing world, and I'm trying to create a project using Sonnet. I've created a "Windows Forms Application (.NET Framework)" with '.NET Framework 4.8'. I've added references Sonnet.dll and SonnetWrapper.dll. I'm using "Example1.cs" with the following code:
```
Example1 ex =new Example1();
ex.Run();
```

However, I'm encoutering an exception: 
```
System.IO.FileNotFoundException
HResult=0x80070002
Message = Unable to load file or assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the specified file.
```

How can I solve the issue? Do I need to add any other references? 
Thank you

**Comment by jhmgoossens** [comment, marked as answer]
Could it be that you are referencing the .NET6 DLL but you need the .NET Framework DLLs?

**Reply by ghost** [reply]
Yes!! Thank you for your fast reply!
